src/main/java/org/iban4j/Iban.java

```diff
@@ -220,6 +224,7 @@ public int hashCode() {
     public final static class Builder {
 
         private CountryCode countryCode;
+        private List<CountryCode> countryCodes;
```

Comment:
I would delete `private CountryCode countryCode;` and replace it with instantiating the build with a singletonList(countryCode).